react/features/conference/actions.web.js

```diff
@@ -36,5 +38,7 @@ export function notifyKickedOut(participant: Object, _: ?Function) { // eslint-d
             titleKey: 'dialog.kickTitle',
             titleArguments: args
         }, NOTIFICATION_TIMEOUT_TYPE.STICKY));
+
+        setTimeout(() => dispatch(maybeRedirectToWelcomePage()), NOTIFICATION_TIMEOUT.LONG);
```

Comment:
I think we should instead use an alert dialog with an ok button which performs the redirection upon click. We might also need to dispatch `readyToClose` I think.